Permanent/ViewControllers/FileDetailsViewController.swift

```diff
@@ -403,6 +413,20 @@ extension FileDetailsViewController: UICollectionViewDelegateFlowLayout {
             
             navigationController?.setViewControllers([fileDetailsVC], animated: false)
         }
+        
+        if currentCellType == .location {
+            let locationSetVC = UIViewController.create(withIdentifier: .locationSetOnTap, from: .main) as! LocationSetViewController
+            locationSetVC.delegate = self
+            locationSetVC.file = file
+            if getAddressString() != "(tap to set)" {
+                locationSetVC.locationDetails = getAddressString()
```

Comment:
you shouldn't have to pass this parameter after you move getAddressString to the ViewModel.
Please delete all references to it afterwards

Reply:
Removed "file" variable from LocationSetViewController.